lib/foreman_scap_client/client.rb

```diff
@@ -36,7 +36,14 @@ def scan_command
                                else
                                  ''
                                end
-      "oscap xccdf eval #{fetch_remote_resources} #{profile} #{tailoring_subcommand} --results-arf #{results_path} #{config[@policy_id][:content_path]}"
+      # OpenSCAP 1.3.6 and newer requires the `--local-files` option to use local copies of remote SDS components
+      version = `rpm -q openscap`.split('-')[1]
+      if version >= '1.3.6' and not config[:fetch_remote_resources]
+        local_files = '--local-files /root'
+      else
+        local_files = ''
+      end
```

Comment:
Even though it can't be used here, we have in Foreman itself one macro for safe version comparison: https://github.com/theforeman/foreman/blob/develop/app/services/foreman/renderer/scope/macros/base.rb#L350

You can use same logic here:
```suggestion
      local_files = if Gem::Version.new(version) >= Gem::Version.new('1.3.6') && !config[:fetch_remote_resources]
                      '--local-files /root'
                    else
                      ''
                    end
```